packages/gatsby-theme-adr/src/components/SEO.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren } from 'react';
+import { useSiteMetadata } from '../hooks/use-site-metadata';
+
+export type SEOProps = PropsWithChildren<{
+  title?: string;
+  description?: string;
+  pathname?: string;
+}>;
+
+const SEO = ({ title, description, pathname, children }: SEOProps) => {
+  const {
+    siteUrl,
+    title: defaultTitle,
+    description: defaultDescription,
+    twitterUsername,
+    image,
+  } = useSiteMetadata();
+
+  const seo = {
+    title: title || defaultTitle,
+    description: description || defaultDescription,
+    image: image ? `${siteUrl}${image}` : '',
+    url: `${siteUrl}${pathname || ``}`,
+    twitterUsername,
+  };
+
+  return (
+    <>
+      <title>{seo.title}</title>
+      <meta name="description" content={seo.description} />
+      <meta name="image" content={seo.image} />
+      <meta name="twitter:card" content="summary_large_image" />
```

Comment:
Does it make sense to add the Open Graph tags too?

Reply:
:thinking: maybe that **is** a rabbit hole not worth entering. I will remove the Twitter specific ones so we don't favor any particular social network.

Reply:
Done.

Reply:
Nice :)